recipes/ucsc-bigwigsummary/meta.yaml

```diff
@@ -25,29 +24,33 @@ requirements:
     - make
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - binutils # [linux]
+    - binutils  # [linux]
   host:
     - libpng
     - libuuid
     - mysql-connector-c
-    - openssl
+    - mariadb-connector-c
```

Comment:
@mencian  does it really need both `mysql-connector-c` and `mariadb-connector-c` ?